### PR TITLE
Separate WACC and UWA users

### PR DESCRIPTION
**Pull request by yasgreen93**

### CODE REVIEW

**Comment by Jamiewarb** [comment]
Think it may be better to have a wacc.js which imports all the WACC stuff, and then you import that on to the client:

`this.$whirli.wacc.users()` rather than `this.$whirli.waccUsers()`

Then Bella can just inject `this.$whirli.wacc` as `this.$wacc`

**Comment by yasgreen93** [comment]
> Think it may be better to have a wacc.js which imports all the WACC stuff, and then you import that on to the client:
> 
> `this.$whirli.wacc.users()` rather than `this.$whirli.waccUsers()`
> 
> Then Bella can just inject `this.$whirli.wacc` as `this.$wacc`

So @Jamiewarb the folder structure should be like so?
```
/Resources
  /wacc
    /Auth
    /User
    .....
  /uwa
    /Auth
    /User
    .....
```
??

File: src/Client.ts [hunk 1]
```diff
@@ -1,20 +1,22 @@
 import ApiOptions from './Interfaces/ApiOptions';
 import HttpClient from './Interfaces/HttpClient';
 import resources from './Resources';
-import Auth from './Resources/Auth/auth';
-import Orders from './Resources/Orders/orders';
+import Search from './Resources/wacc/Search/search';
+import ReturnOrders from './Resources/wacc/Returns/returnOrders';
```
Comment (Jamiewarb) [inline src/Client.ts]: Do we still need these here if they're WACC resources? Shouldn't that all be covered by the WACC object?

File: src/Resources/Search/search.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import AbstractResource from '../AbstractResource';
 
 export default class Search extends AbstractResource {
     initialise(): void {
-        this.resourcePath = 'wacc/search';
+        this.resourcePath = 'search';
```
Comment (Jamiewarb) [inline src/Resources/Search/search.ts]: Does this file belong inside of the WACC directory now?

**Pull request merged**